packages/hash/shared/src/entityStore.ts

```diff
@@ -178,5 +176,9 @@ export const createEntityStore = (
     );
   }
 
+  for (const [draftId, draftEntity] of Object.entries(draftData)) {
+    draft[draftId] ??= draftEntity;
+  }
+
```

Comment:
This looks like a candidate for `Object.assign`. Example:
```js
draft = Object.assign({}, draftData, draft);
```
Though if you need to mutate `draft` I guess this is not an option.

Reply:
This is using immer, so mutating `draft` is necessary and by design. But thank you!